fern/pages/instruments-guide/futures/tier-2/pnut-usd-perp.mdx

```diff
@@ -11,7 +11,7 @@ export const orderSizeIncrement = "1 PNUT";
 export const minOrderValue = "50 USD";
 export const maxOrderSize = "170,000 PNUT";
 export const maxOpenOrders = "50";
-export const positionLimit = "5,800,000 PNUT";
+export const positionLimit = "7,734,300 PNUT";
```

Comment:
@varunjain15 - could you double check this one too?  doc says 5,800,000

Reply:
fixed